dist/scripts/setup-data-dir.sh

```diff
@@ -49,27 +51,37 @@ create_rkt_file() {
     chgrp rkt "${file}"
 }
 
-make_rkt_directory "${datadir}" 2750
-make_rkt_directory "${datadir}/tmp" 2750
+getent group rkt-admin || groupadd --force --system rkt-admin
+getent group rkt || groupadd --force --system rkt
 
-make_rkt_directory "${datadir}/cas" 2770
-make_rkt_directory "${datadir}/cas/db" 2770
+if which systemd-tmpfiles; then
+    systemd-tmpfiles --create /usr/lib/tmpfiles.d/rkt.conf
+    exit 0
```

Comment:
Please omit the 0 here, so that it will just exit with `systemd-tmpfiles` return value. It shouldn't matter in most of the cases anyway, but I find it cleaner.